### ISSUE TITLE
Feature request: Timeout

### ISSUE THREAD

**Issue by happiehappie** [lypiut/WhereAmI#13]
Possible to request for a timeout property?

**Comment by hashimkhanbaloch** [comment]
Yes,  Possible to request time out Property in this scenes to remove update  line and add locally notification to Lant , long Updation .
